front/app/containers/Admin/projects/components/ProjectRow/GroupsTag.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { isNilOrError } from 'utils/helperUtils';
+import { colors } from 'utils/styleUtils';
+import { StyledStatusLabel } from '.';
+import ConditionalWrapper from 'components/ConditionalWrapper';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+import useProjectGroups from 'hooks/useProjectGroups';
+import Link from 'utils/cl-router/Link';
+import { adminProjectsProjectPath } from 'containers/Admin/projects/routes';
+
+interface Props {
+  projectId: string;
+  userCanModerateProject: boolean;
+}
+
+const GroupsTag = ({ projectId, userCanModerateProject }: Props) => {
```

Comment:
Could be good to add some unit tests to at least some of these new components.

Reply:
They're not new components, just extracted from an existing component so chances of having broken something are minimal. I also tested them manually. That said, if I have time tomorrow, I woud love to create some unit tests in a separate PR.

Reply:
That'd be great!